content/how-to/dedicated-hosting/manage-local-users-in-the-myrackspace-portal/index.md

```diff
@@ -1,44 +1,52 @@
+title: Manage Local Users in the MyRackspace Portal
 type: article
 created_date: '2019-12-12'
 created_by: Chad Sterling
-last_modified_date: '2021-07-07'
-last_modified_by: Cat Lookabaugh
+last_modified_date: '2022-11-11'
+last_modified_by: Asmita Nakwa
 product: Dedicated Hosting
 product_url: dedicated-hosting
 ---
 
 This article explains how to use a ticket template to manage local users.
 
-### Create a ticket in the MyRackspace portal
+### Create a Ticket in the MyRackspace portal
 
-1. Log in to the [MyRackspace portal](https://login.rackspace.com/login) with
+**Step 1.** Log in to the [MyRackspace portal](https://login.rackspace.com/login) with
    your username and password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="Enter login credentials" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement1.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+**Step 2.** In the top navigation bar, choose **Support** > **Support Center**.
 
-4. Select the **Subject** field.
+   <img width="941" alt="Select support center from drop down menu" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement2.png">
 
-5. From the drop-down menu, select **Local User Management.**
+**Step 3.** Under **Common Request** menu, click **See All Templates**. It displays list of all templates.
 
-   {{<image src="localuser1.png" alt="" title="">}}
+   <img width="958" alt="Displays the list of templates" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement3.png">
 
-6. Select the affected **Device(s)** and **Action Type**.
+**Step 4.** From the **Common Request** menu, select **Local User Management**.
 
-#### Create a user
+   <img width="960" alt="Select local usermanagement from the templates" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement4.png">
 
-1. Enter the **Username**, **Password**, and **Full Name** of the user you want
+**Step 5.** In **Issue Details** section you can select the appropriate **Device(s)** and **Action Type**.
+
+   <img width="644" alt="Select the device and action types from Issue Details section" src="/support/how-to/manage-local-users-in-the-myrackspace-portal/localusermanagement5.png">
+
+On each chosen server, the action you choose next will be executed. The details of each action are described below:   
+
+**Create a User on the Selected Device**
+
+**Step 1.** Enter the **Username**, **Password**, and **Full Name** of the user you want
    to create on the selected devices.
 
    The **Password** should be at least eight characters, with at least three
    character types (uppercase, lowercase, numeric, and symbol).
 
-2. You can also assign the following attributes to the user:
+**Step 2.** You can also assign the following attributes to the user:
```

Comment:
Is this list supposed to render like this? Was it like this before? Can we fix it?
![Screen Shot 2023-02-01 at 3 03 02 PM](https://user-images.githubusercontent.com/14296817/216162857-216c24fe-a055-47a9-8fb0-ce998ceddef8.png)